configure.ac

```diff
@@ -437,6 +437,14 @@ else
     AC_DEFINE_UNQUOTED([unlikely(x)], [(x)], [gcc branch optimization])
 fi
 
+if test "${GCC}" = "yes"; then
+    AC_DEFINE([__always_unused], [__attribute__((unused))], [gcc unused attribute])
```

Comment:
> Hmm, OK musl libc uses __unused itself in various places... let's rename __unused to __always_unused

If there is a collision problem i suggest to use `netdata` prefix, like `__netdata_used`.

`__always_unused` doesnt look goot to me, because `__unused` already means always. And if we have `__always_unused`,  then `__unused` means something else.

Reply:
musl uses __unused variables for structure padding (others generally use __pad or somesuch), so you would never generally see or use __unused from it and they are certainly not the same thing here.

But if you're not keen on the naming, here are some alternatives

__a_unused / __m_unused
UNUSED / MAYBE_UNUSED

other suggestions welcome.

Reply:
@mfundul I will convert the tabs to spaces (the blasphemy! ;), I need to sort out a vim config for netdata...)

Reply:
> __a_unused / __m_unused

Well, i think if you can make something descriptive then make it. `a` and `m` is not descriptive at all. So `always/maybe` > `a/m`.

To be honest i dont have strong opinion on that, always/maybe is ok.